step-capstone/src/components/Firebase/firebase.js

```diff
@@ -69,6 +72,51 @@ class Firebase {
     return tripListRef.add(trip);
   }
 
+  async deleteTrip(reference) {
+    const tripRef = this.db.doc(reference);
+    return new Promise((resolve, reject) =>
+     {
+      tripRef.delete().then(() => {
+        console.log("Trip successfully deleted!");
```

Comment:
Should we remove the console.logs? Theres some others in this file too, not sure if that was intentional or if you forgot to remove them

Reply:
Oh, this one is intentional? I thought this was similar to "You have logged in" and sort of thing? Should I delete it?